packages/auth/src/platform_browser/strategies/phone.test.ts

```diff
@@ -47,6 +47,11 @@ import {
 use(chaiAsPromised);
 use(sinonChai);
 
+//interfaces added to provide typescript support for webopt autofill
```

Comment:
Is this needed in the test fil?

Reply:
Yes, I believe this is needed for us to use OTPCredential in the unit tests. It can be removed once typescript support is added for the WebOTP API.